docs/use-cases/headless-cms/security.md

```diff
@@ -0,0 +1,109 @@
+---
+description:
+  This guide covers best practices for security, authentication, and permissions within the context of Headless CMS.
+tags: []
+skill_level:
+directus_version:
+author_override:
+author: Bryant Gillespie
+---
+
+# Security in Headless CMS
+
+> {{ $frontmatter.description }}
+
+[Headless CMS](/use-cases/headless-cms/introduction) as an architecture choice is generally more secure than Traditional
+CMS because your content is separated from the presentation layer. Given that users don't directly interact with a
+server to load a page, there is a reduced ability for malicious actors to attack the website. However, it is still very
```

Comment:
```suggestion
server to construct a page, there is a reduced ability for malicious actors to attack the website. However, it is still very
```

Reply:
A server does load the page. The fact it served up a static file is where the distinction is.